### ISSUE TITLE
[Feature Request] hCaptcha support

### ISSUE THREAD

**Issue by ItzExotical** [FOSSBilling/FOSSBilling#1658]
**Is your feature request related to a problem? Please describe.**
I'm always frustrated when amazing software like FOSSBilling does not support hCaptcha.
hCaptcha is way better at stopping bots by giving unique challenges that are hard for bots every time.
I use hCaptcha in all my apps, and it is very easy to implement. So please, for the love of God, implement it. 😛

**Describe the solution you'd like**
Support for hCaptcha in FOSSBilling for actions such as register, login, ticket submit, etc.

**Describe alternatives you've considered**
No. hCaptcha for the win.

**Additional context**
No. 😛

**Comment by John-S4** [comment]
Cloudflare Turnstile was another proposed option for an update to the captcha implementation in FOSSBilling - #308 but this is a better forward option, so that issue has been closed in favour of this one.

**Comment by John-S4** [comment]
To be implemented after modularization of the captcha system - see #1662

**Comment by BelleNottelling** [comment]
I don't think we need an open issue for each specific potential captcha implementation.
Closing in favor of #1662
